monero-outreach-docs/translations/fr/archive/we_accept_monero_fr.md

```diff
@@ -0,0 +1,21 @@
+
+# On accepte Monero 
+
+**« Êtes-vous fatigué de prier pour l’adoption des cryptos ? Assez d’attendre pour que Monero sorte de l’ombre ? Soyez de ceux qui acte pour l’écosystème soit tel que vous le voudriez. »  - 20 Mai 2020**
+
+Nous souhaiterions vous inviter à nous rejoindre afin de perpétrer une bonne vielle « campagne de harcèlement » envers vos commerces favoris qui n’acceptent pas encore XMR.  La « campagne de harcèlement » la plus impressionnante et la mieux réussi de tous les temps se devait être écrite par George Lois. En 1982 les compagnies proposant la télévision par câble n’avaient aucun intérêt pour une chaine de musique 24 heures sur 24; elles ne pensaient pas que cela marcherait. George Lois eu cette idée et filma une série de publicités avec des stars du rock et qui disaient aux téléspectateurs d’appeler leur compagnie de télévision par câble en leur demandant « Je veux ma MTV! ». Le reste fait partie de l’histoire [youtu.be/vAyOcWpm3QU](https://youtu.be/vAyOcWpm3QU?t=71).
+
+La mission, si vous l’acceptez, est de contacter votre commerce favori et de lui demander s’il a déjà considéré d’accepter Monero, car cela serait votre méthode préférée de paiement. Il se peut qu’ils disent oui, il se peut qu’ils disent non, mais au moins vous les faites penser à cette possibilité. Ce ne sera pas la dernière fois qu’ils entendrons parler de Monero, mais cela pourraient être leur première fois.
+
+La sagesse populaire en marketing dit qu’une personne doit voir un message sept fois avant de prendre conscience dudit message. C’est une règle approximative en publicité, mais on espère encourager une discussion avec les commerces. Une bonne conversation vaut bien sept publicités atteignant leur but. La recherche a découvert que les gens qui connaissent une organisation ont cinq fois plus de chances d’en avoir un avis favorable que ceux qui ne la connaisse pas. En d’autres termes, plus vous parlez de Monero, plus les gens vont l’apprécier.
+
+On a aussi créé cette brochure de la taille d’un billet d’un dollar que vous pouvez imprimer et mettre dans votre portefeuille lorsque vous êtes de sortie. Cette chose se focalise sur trois aspects uniques de Monero qui sont très important pour les propriétaires de commerces et les invite à en apprendre plus sur la page [Monero Merchant Services](https://www.monerooutreach.org/merchants).
+
+Vous pouvez télécharger un fichier prêt à être imprimé depuis [WhenYouAcceptMonero-en-WalletBuckslip.pdf](https://static.monerooutreach.org/img/MoneroMerchants/WhenYouAcceptMonero-en-WalletBuckslip.pdf). Bonne Chance !
+
+
+### _En apprendre plus_
+
+- [Une introduction sur comment aider Monero](https://www.monerooutreach.org/stories/getting-started-helping-monero.html)
+- [Un guide essentielle pour un Marketing de type guérilla pour Monero](https://www.monerooutreach.org/guerrilla-toolkit.html)
```

Comment:
Hey, noticed we are missing the link to the French version. 

Could you please change the link:

https://www.monerooutreach.org/boite-outils-marketing-guerilla.html

:)

Reply:
Ok. I changed it. 
However, I realise that https://www.monerooutreach.org/stories/getting-started-helping-monero.html was also translated (by Hsterofemia) and reviewed (by myself) and merged (see: https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/us/270). However I do not see any link to the French version of the website. Would be good if this one was also integrated into the website so that I can provide a French link

Reply:
Yes, this hasn't been added to the site yet. We have to push it. We have been a little busy, sorry for the delay.